src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
```

Comment:
You should avoid using color codes directly in the code. It's better to use variables for colors as you did in line 11. This makes it easier to change the color scheme of the website in the future.